setup.py

```diff
@@ -10,7 +10,7 @@
     author_email="info@zyte.com",
     url="https://github.com/scrapy-plugins/scrapy-zyte-api",
     packages=["scrapy_zyte_api"],
-    install_requires=["zyte-api>=0.1.2", "scrapy>=2.6.0"],
+    install_requires=["zyte-api>=0.3.0", "scrapy>=2.6.0"],
```

Comment:
Note to self: bump2version is going to screw this up as well.

Reply:
of course, I tried to make a release and forgot about it :)